packages/components/src/Chip/components/ChipDismissible/ChipDismissible.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Chip } from "../../Chip";
+import { ChipProps } from "../../Chip.types";
+import { Icon } from "../../../Icon";
+
+export function ChipDismissible(props: ChipProps) {
```

Comment:
Do we need to narrow down that props? I wonder what happens when you add a children on the usage 🤔 . Would it override the line `9`?

Reply:
![Screenshot 2023-10-11 at 7 50 47 PM](https://github.com/GetJobber/atlantis/assets/145565743/544407a7-e59f-49f1-9ef9-e9bb17e94442)

I was curious myself, turns out it's fine to pass in as-is. My image React treats `children` a bit differently than a normal prop.

Reply:
Oh also, this prompted me to bring a Dismissible and Selectable chip into the examples to test this. So this comment made the PR even better. :)